packages/devtools_app/lib/src/debugger/flutter/debugger_controller.dart

```diff
@@ -3,13 +3,21 @@
 // found in the LICENSE file.
 
 import 'dart:async';
+import 'dart:convert';
 
 import 'package:flutter/foundation.dart';
 import 'package:vm_service/vm_service.dart';
 
 import '../../auto_dispose.dart';
+import '../../core/message_bus.dart';
 import '../../globals.dart';
 
+// Note: don't check this in enabled.
```

Comment:
This doesn't seem debugger specific. Maybe we should add this to 
VmServiceWrapper?

Reply:
Will do -